src/test/ui/thread-local-not-in-prelude.rs

```diff
@@ -3,7 +3,7 @@
 
 extern crate std;
 
-std::thread_local!(static A: usize = 30);
+std::thread_local!(static _A: usize = 30);
```

Comment:
Wow, so the unused variable lint wasn't triggering before?

Did you only have to make this change before you fixed the lint logic, though?

Reply:
Oops; forgot to revert this -- should be done now when I push again (after figuring out the other problem...).